docs/sources/panels-visualizations/visualizations/geomap/index.md

```diff
@@ -250,6 +274,112 @@ The GeoJSON layer allows you to select and load a static GeoJSON file from the f
+- **Fill opacity** configures the opacity of the route.
+- **Text label** configures a text label for each route.
+- **Arrow** sets the arrow styling to display along route, in order of data.
+  - **None**
+  - **Forward**
+  - **Reverse**
+- **Display tooltip** allows you to toggle tooltips for the layer.
+
+{{< figure src="/media/docs/grafana/geomap-route-layer-arrow-size-9-4-0.png" max-width="1200px" caption="Geomap panel Route arrows with size" >}}
+
+### More information
+
+- [**Extensions for OpenLayers - Flow Line Style**](http://viglino.github.io/ol-ext/examples/style/map.style.gpxline.html)
+
+## Photos layer (Beta)
+
+The Photos layer renders a photo at each data point.
+
+{{< figure src="/static/img/docs/geomap-panel/geomap-photos-9-3-0.png" max-width="1200px" caption="Geomap panel Photos" >}}
+
+### Options
+
+- **Image Source Field** select a string field containing image data in either of the following formats
+  - **Image URLs**
+  - **Base64 encoded** image binary ("data:image/png;base64,...")
+- **Kind** select the frame style around the images
+  - **Square**
+  - **Circle**
+  - **Anchored**
+  - **Folio**
+- **Crop** toggle if the images are cropped to fit
+- **Shadow** toggle a box shadow behind the images
+- **Border** set the border size around images
+- **Border color** set the border color around images
+- **Radius** set the overall size of images in pixels
+- **Display tooltip** allows you to toggle tooltips for the layer.
+
+{{< figure src="/static/img/docs/geomap-panel/geomap-photos-options-9-3-0.png" max-width="1200px" caption="Geomap panel Photos options" >}}
+
+### More information
+
+- [**Extensions for OpenLayers - Image Photo Style**](http://viglino.github.io/ol-ext/examples/style/map.style.photo.html)
+
+## Network layer (Beta)
+
+The Network layer renders a network graph. This layer supports the same [data format supported by the node graph visualization]({{< relref "../node-graph/#data-api" >}}) with the addition of [geospatial data]({{< relref "#location">}}) included the nodes data. The geospatial data is used to render the nodes on the map.
+
+{{< figure src="/media/docs/grafana/screenshot-grafana-10-1-geomap-network-layer-v2.png" max-width="750px" caption="Geomap network layer" >}}
+
+### Options
+
+- **Arrow** sets the arrow styling to display for each edge.
+  - **None**
+  - **Forward**
+  - **Reverse**
+  - **Both**
+- **Show legend** allows you to toggle the legend for the layer. **Note:** The legend currently only supports node data.
+- **Display tooltip** allows you to toggle tooltips for the layer.
+
+#### Node styles
+
+- **Node Size** configures the size of the node. The default is `Fixed size`, which makes all node sizes the same regardless of the data points. However, there is also an option to scale the nodes to the corresponding data points. `Min` and `Max` node size has to be set such that the nodes can scale within this range.
```

Comment:
I don't see this layer in the test instance I'm in, but please confirm that these UI labels are actually "Node Size", "Node Shape", and "Node Color" vs just "Size", "Shape", and "Color".